lib/avoid_imports_from_current_application.ex

```diff
@@ -7,15 +7,15 @@ defmodule Nicene.AvoidImportsFromCurrentApplication do
 
   @explanation [check: @moduledoc]
 
-  use Credo.Check, base_priority: :high, category: :refactoring
+  use Credo.Check, base_priority: :high, category: :refactoring, param_defaults: [namespaces: []]
 
   @doc false
   def run(source_file, params \\ []) do
     issue_meta = IssueMeta.for(source_file, params)
 
     namespaces =
       params
-      |> Keyword.get(:namespaces, [])
```

Comment:
Shouldn't that be 
```elixir
|> Params.get(:namespaces, __MODULE__)
```
instead? The variant with (keyword) list seems to be deprecated and the current version won't use `param_defaults` AFAIU

Reply:
oops yeah good spot, that's a typo on my part

Reply:
fixed in be60842ab7bbbc30e71bedb0d821a2b747f7a299